src/classes/frPledge.cls

```diff
@@ -49,6 +49,18 @@ public with sharing class frPledge {
                                         ];
         return possiblePledges.size() > 0 ? possiblePledges.get(0) : null;
     }
+
+    public static Pledge__c findActiveAcc(Id accountId, Date closeDate) {
+        List<Pledge__c> possiblePledges = [SELECT Id FROM Pledge__c 
+                                         WHERE Supporter_PA__c = :accountId 
```

Comment:
The lookup field referenced here has not been added to the repository so this would have errors deploying

Reply:
Added Account lookup for Pledge!